### PR TITLE
feat(groups): clearer join/owner status and less prominent leave button

### LINKED ISSUE #8872: Remove Leave Group from title menu
Currently, the first thing you see when visiting the group page is "Leave Group" call to action button in a place reserved for things like "Add a blog" and "Join group". Perhaps that would be a more logical place for "Start a discussion", at least on the community site. Curently one has to click around before figuring out how to add a discussion topic.
"Leave group" should probably go into "My status" menu instead.

### PR DESCRIPTION
**Pull request by mrclay**
Joining/creating a group leaves a title button that indicates “Joined” or “Owned” and places the Leave button in a dropdown child menu, along with a group notifications link.

Fixes #8872

https://drive.google.com/file/d/15LpBoqHySSgA8Hw4j0OXNr5QaH3KD55Z/view?usp=sharing

### CODE REVIEW

**Comment by jdalsem** [comment]
screenshot?

File: mod/notifications/start.php [hunk 1]
```diff
@@ -116,6 +118,40 @@ function _notifications_page_menu($hook, $type, $return, $params) {
...
+
+	$user = elgg_get_logged_in_user_entity();
+	if (!$user) {
+		return;
+	}
+
+	$items = $hook->getValue();
+	/* @var ElggMenuItem[] $items */
+
+	foreach ($items as $item) {
```
Comment (jdalsem) [inline mod/notifications/start.php]: why looping this? why not add directly as a child?

Comment (mrclay) [reply mod/notifications/start.php]: I don't see any other way other than adding in the groups plugin, but I wanted to keep this in notifications.

**Comment by mrclay** [comment]
@jdalsem Video in the description

File: mod/notifications/start.php (hunk 1 above)
Comment (mrclay) [reply mod/notifications/start.php]: If there's a better way let me know.

**Pull request merged**